packages/paste-website/src/components/paste-mdx-provider/index.tsx

```diff
@@ -83,13 +83,13 @@ const MDXPoviderComponents = {
   strong: (props: React.ComponentProps<'strong'>): React.ReactElement => <strong {...props} />,
   del: (props: React.ComponentProps<'del'>): React.ReactElement => <del {...props} />,
   hr: (props: SeparatorProps): React.ReactElement => (
-    <Separator {...props} orientation="horizontal" verticalSpacing="space70" />
+    <Separator {...props} orientation="horizontal" verticalSpacing="space100" />
```

Comment:
I updated the Figma specs so we're using `space100` on top/bottom of the Separator